ops/testing.py

```diff
@@ -339,18 +338,17 @@ class TestCharm(self._charm_cls):  # type: ignore
         self._charm = TestCharm(self._framework)  # type: ignore
 
     def begin_with_initial_hooks(self) -> None:
-        """Called when you want the Harness to fire the same hooks that Juju would fire at startup.
+        """Have the Harness fire the same hooks that Juju would fire at startup.
```

Comment:
Nice simplification. Probably could even do this (as all of these are things the `Harness` does):

```suggestion
        """Fire the same hooks that Juju would fire at startup.
```

Reply:
I had this originally, actually, and I second-guessed myself that perhaps it was important to keep the harness reference there, even though you're in the harness class. I like this better :)